infra/resources/prod/function_ioweb_profile.tf

```
@@ -0,0 +1,300 @@
+      FUNCTIONS_APP_CLIENT_BASE_URL = "https://io-p-itn-auth-profile-fn-02.azurewebsites.net"
+
+      // -------------------------
+      // Hub Spid Login for ioweb config
+      // -------------------------
+      HUB_SPID_LOGIN_API_KEY         = data.azurerm_key_vault_secret.spid_login_api_key.value
+      HUB_SPID_LOGIN_CLIENT_BASE_URL = "https://io-p-weu-ioweb-spid-login.azurewebsites.net"
+
+      // -------------------------
+      // Audit Logs config
+      // -------------------------
+      AUDIT_LOG_CONNECTION_STRING = data.azurerm_storage_account.immutable_spid_logs_storage.primary_connection_string
+      AUDIT_LOG_CONTAINER         = local.immutable_audit_logs_container_name
+    }
+  }
+}
+
+resource "azurerm_resource_group" "function_web_profile_rg" {
+  name     = "${local.project}-${local.domain}-${local.function_ioweb_profile.name}-rg-01"
+  location = local.location
+
+  tags = local.tags
+}
+
+module "function_web_profile" {
+  source = "github.com/pagopa/dx//infra/modules/azure_function_app?ref=ab26f57ed34a614fd3fa496c7b521be9ecc88e1b"
+
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    location        = local.location
+    domain          = local.domain
+    app_name        = local.function_ioweb_profile.name
+    instance_number = "01"
+  }
+
+  resource_group_name = azurerm_resource_group.function_web_profile_rg.name
+  health_check_path   = "/api/v1/info"
+  node_version        = 20
+  app_service_plan_id = data.azurerm_app_service_plan.shared_plan_itn.id
+
+  subnet_cidr   = local.cidr_subnet_fn_web_profile
+  subnet_pep_id = data.azurerm_subnet.private_endpoints_subnet.id
+
+  subnet_service_endpoints = {
+    web     = true,
+    storage = true
+  }
+
+  private_dns_zone_resource_group_name = data.azurerm_resource_group.rg_common.name
+
+  virtual_network = {
+    name                = data.azurerm_virtual_network.itn_common.name
+    resource_group_name = data.azurerm_virtual_network.itn_common.resource_group_name
+  }
+
+  app_settings = merge(
+    local.function_ioweb_profile.app_settings
+  )
+  slot_app_settings = merge(
+    local.function_ioweb_profile.app_settings
+  )
+
+  application_insights_connection_string = data.azurerm_application_insights.application_insights.connection_string
+
+  action_group_id = azurerm_monitor_action_group.error_action_group.id
+
+  tags = local.tags
+}
+
+module "function_web_profile_autoscale" {
+  depends_on = [azurerm_resource_group.function_web_profile_rg, module.function_web_profile]
+  source     = "github.com/pagopa/dx//infra/modules/azure_app_service_plan_autoscaler?ref=ab26f57ed34a614fd3fa496c7b521be9ecc88e1b"
+
+  resource_group_name = azurerm_resource_group.function_web_profile_rg.name
+  target_service = {
+    function_app_name = module.function_web_profile.function_app.function_app.name
+  }
+
+  scheduler = {
+    normal_load = {
+      minimum = 3
+      default = 10
+    },
+    maximum = 30
```

```suggestion
    normal_load = {
      minimum = 2
      default = 2
    },
    maximum = 30
```
if we want to reproduce the actual config this would be the right setting

---

Minimum should be `3` for the right zone redundancy configuration.

---

fixed in [f2e8ac2](https://github.com/pagopa/io-auth-n-identity-domain/pull/126/commits/f2e8ac239fd165a93947dc378b760cf44814f577)